src/core/scene/a-scene.js

```diff
@@ -169,7 +169,7 @@ var AScene = module.exports = registerElement('a-scene', {
         if (!camera || !canvas) { return; }
 
         // Update canvas.
-        if (!isMobile) {
+        if (!isMobile && !canvas.dataset.aframeProvided) {
```

Comment:
i'm not sure what you mean. we only want to run this block if the canvas was not provided, correct?